src/main/java/org/apache/commons/text/RandomStringGenerator.java

```diff
@@ -183,9 +183,11 @@ public Builder filteredBy(final CharacterPredicate... predicates) {
          * @since 1.2
          */
         public Builder selectFrom(final char... chars) {
-            characterList = new ArrayList<>();
-            for (final char c : chars) {
-                characterList.add(c);
+            if(chars != null) {
```

Comment:
Checkstyle issues in the PR (you can run `mvn` locally and that should do the same as GH Actions).

```bash
 Error:  src/main/java/org/apache/commons/text/RandomStringGenerator.java:[244,13] (whitespace) WhitespaceAfter: 'if' is not followed by whitespace.
```